geoapi/app.py

```diff
@@ -62,7 +62,7 @@ def handle_streetview_auth_exception(error: Exception):
 def handle_streetview_limit_exception(error: Exception):
     return {'message': 'Exceed concurrent streetview publish limit'}, 403
 
-
+# ensure SQLAlchemy sessions are properly closed at the end of each request.
```

Comment:
this is how the db sessions are ensured to be closed properly after each request. so not certain but something in celery might be needed.